### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 10

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#336] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324779-c04e20ff-1eab-42ab-b5ad-8591c836f303.png)

**Comment by danieldinella** [comment]
1. A ≠ B o A = B = 0
2. A = 1 e B = 0
3. A = 1 e B = 0

**Comment by Exyss** [comment]
1. Qualsiasi assegnamento che non sia A = B = 1
2. A = 1, B = 0
3. A = 1, B = 0

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217808796-ebb662e3-dcca-47f9-a135-af55a879315a.png)
